### ISSUE TITLE
AAAAAA automatic value for an empty example

### ISSUE THREAD

**Issue by b2p-fred** [rapi-doc/RapiDoc#684]
When I set an empty value for the example value of a field, value is set automatically with `AAAAAA`. The same behaviour is met when no example is set.

In my specification: 
```
          content:
            application/json:
              schema:
                type: 'object'
                required:
                  - lat
                  - lng

                properties:
                  line1:
                    description: 'First line'
                    type: string
                    maxLength: 255
                    example: '127, Avenue Joseph Pierre Boitelet'
                  line2:
                    description: 'Second line'
                    type: string
                    maxLength: 255
                    example: ''
                  postalCode:
                    description: 'Zip code'
                    type: string
                    minLength: 3
                    maxLength: 10
                    example: '84300'
```

The Json example:
```
{
  "line1": "127, Avenue Joseph Pierre Boitelet",
  "line2": "AAAAAA",
  "postalCode": "84300",
  "city": "Cavaillon",
  "country": "AAAAAA",
  "lat": 43.8366045,
  "lng": 5.0407814
}
```

As you see, the `line2` field is set with `AAAAAA`

**Comment by mrin9** [comment]
will look into it

**Comment by mrin9** [comment]
fixed by #759
